mmv1/third_party/terraform/services/securitycenter/resource_scc_organization_big_query_export_config_test.go

```diff
@@ -13,13 +18,21 @@ func TestAccSecurityCenterOrganizationBigQueryExportConfig_basic(t *testing.T) {
 	t.Parallel()
 
 	randomSuffix := acctest.RandString(t, 10)
-	dataset_id := "tf_test_" + randomSuffix
+	datasetID := "tf_test_" + randomSuffix
 	orgID := envvar.GetTestOrgFromEnv(t)
 
+	// Run cleanup before the test starts
+	ctx := context.Background()
+	projectID := envvar.GetTestProjectFromEnv()
+	err := cleanupOrganizationBigQueryDatasets(ctx, "tf_test_", projectID)
```

Comment:
This seems to be project-scoped, not org scoped?

Reply:
BigQuery datasets are project-scoped, even though the test configures an organization-level, the actual export targets a BigQuery dataset, which is scoped to a specific project (not the organization). Therefore, managing datasets for cleanup requires a project ID.

Reply:
Per above- let's remove this action.